### ISSUE TITLE
[BUG] Getting throttling function problem

### ISSUE THREAD

**Issue by sleepingcat4** [pytube/pytube#1960]
I'm getting this error: Error processing link https://www.youtube.com/watch?v=TCH6gbDo2zc: get_throttling_function_name: could not find match for multiple

can someone help me out?

**Comment by aglpy** [comment]
I just have the same problem, it seems a change on youtube as the code and library hadn't been updated since some time ago.

**Comment by EJ-AIpreneur** [comment]
Yea, I get the same problem, I would just wait a few hours for the library to be updated for the YouTube change.

**Comment by ArtemBernatskyy** [comment]
I know it's not helping but +1

**Comment by gagaos** [comment]
Yes, i have the same problem. It worked one time, but not animore.

**Comment by gagaos** [comment]
A solution seam to be here: https://github.com/pytube/pytube/issues/1954
